src/ui/molecules/ExpandingEmailField.test.tsx

```diff
@@ -0,0 +1,165 @@
+import '../../../test-utils/i18n-mock';
+import React, { useState, Fragment, ChangeEvent } from 'react';
+import { cleanup, render, RenderResult, fireEvent } from '@testing-library/react';
+import ExpandingEmailField from './ExpandingEmailField';
+
+describe('Expanding Email Field', () => {
+    afterEach(cleanup);
+
+    it('should render correctly', () => {
+        expect(
+            (): RenderResult =>
+                render(
+                    <ExpandingEmailField
+                        register={jest.fn()}
+                        name="test"
+                        inputRows={[{ name: '', email: '' }]}
+                        maxRows={1}
+                    />,
+                ),
+        ).not.toThrow();
```

Comment:
shouldn't we also expect something to be visible?

Reply:
This just ensures the file has valid code more than anything else. Visible element tests should be their own test cases but because we're testing interactions with the elements we'd expect that case is tested by design